bin/ch/WScriptJsrt.cpp

```diff
@@ -490,9 +495,39 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             LPCWSTR errorMessage = nullptr;
             size_t errorMessageLength = 0;
 
-            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"message", &messagePropertyId), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, messagePropertyId, &messageProperty), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength), false);
+            LPCWSTR errorName = nullptr;
+            size_t errorNameLength = 0;
+
+            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"name", &namePropertyId), false);
```

Comment:
>  For the sake of output consistency, I would go for the second. 

I see your point. However as per [Error.prototype.toString](http://www.ecma-international.org/ecma-262/6.0/#sec-error.prototype.tostring), the toString() would return `name : message` anyway. Having `toStirng()` call here (instead of manually crafting `name : message`) would be good to make sure we don't break `Error.prototype.toString` in future (or if we do, we consciously update the baselines). Thoughts?